containers/datalab/Dockerfile.in

```diff
@@ -103,7 +104,6 @@ RUN mkdir -p /datalab/web && \
         mkdirp@0.5.1 \
         node-uuid@1.4.7 \
         bunyan@1.7.1 \
-        socket.io@1.4.5 \
         tcp-port-used@0.1.2 \
         node-cache@3.2.0 && \
     cd / && \
```

Comment:
Might it be better to use the CPU version of TensorFlow?
For the GPU version, CUDA toolkit 7.5 and CuDNN v4 need to be installed as well.